.github/workflows/ruff.yml

```diff
@@ -1,22 +0,0 @@
-name: Ruff
```

Comment:
please leave this as is.
it provides nice inline annotations on PRs in case linting fails

Reply:
Ah, gotcha. Then I'll just update its Ruff version, and signpost the code to keep all 3 versions of Ruff in sync 😅

Reply:
`ruff check --output-format=github grapple`

Reply:
On the one hand, it's intimidating to know you're looking at _everything, in real time_. On the other hand, this probably saved me a fair amount of digging. On balance, thank you!

Reply:
'twas pure coincidence! Just happened to check what's new in GitHub